documentation/operators/src/legal/landing-pages.md

```diff
@@ -7,3 +7,328 @@ The entire content of this page is under [Creative Commons Attribution 4.0 Inter
 > Nym Node operators are invited to add their legal findings and suggestions directly through [pull requests](add-content.md).
 
 Exit Gateway landing page is a great and transparent way to prevent possible troubles by providing people with basic facts, links to laws and regulations on a given topic as well as operator's contact info. To inspire each other we share some examples how Nym community handled this issue below.
+
+## Avril 14th Exit Gateways
+
+The following snippet it on the 'assets' path under '[http]' section of your config file
+```
+landing_page_assets_path = '<path to your custom html file>'
+```
+
+Customize it as follows:
+
+- add your favicon logo:
+```
+<link rel="icon" type="image/png" href="">
+```
+
+and your header logo
+```
+<img class="logo" src="<FIXME>">
+```
+
+adding your favicon as base64 encoded image or add it as URl if you're hosting it publicly, i.e.
+
+href="path to your public url"
+or
+href="href="data:image/x-icon;base64,AAABAAMA....""
```

Comment:
I would consider to keep the href syntax lines in `code syntax` as well so it's well separated from the text.

Reply:
Yes, and I'm rewording a bit more to clarify the tasks